descriptors/ModuleDescriptor-template.json

```diff
@@ -368,6 +368,52 @@
         }
       ]
     },
+    {
+      "id": "batch-voucher.batch-voucher-exports",
+      "version": "1.0",
+      "handlers": [
```

Comment:
add "scheduled" and "upload" to moduleDescriptor please

Reply:
Done. I didn't add because there are no `modulePermissions` for these stubs but seems like we still need them.